brainfuck/commands.go

```diff
@@ -0,0 +1,70 @@
+package brainfuck
+
+import "fmt"
+
+type command interface {
+	Execute(mem *memory)
+}
+
+/*
+	Moves pointer in memory struct to right
```

Comment:
`in memory struct`? Why do you use `struct` here? I think you can just remove it and have something like:
`Moves a pointer in memory to right`

The same for the docs in the other commands

Reply:
Fixed